tasks/release_linux.js

```diff
@@ -86,7 +86,7 @@ var packToDebFile = function () {
     packDir.write('DEBIAN/control', control);
 
     // Build the package...
-    childProcess.exec('fakeroot dpkg-deb -Zxz --build ' + packDir.path() + ' ' + debPath,
+    childProcess.exec('fakeroot dpkg-deb -Zxz --build ' + packDir.path().replace(/\s/, '\\ ') + ' ' + debPath.replace(/\s/, '\\ '),
```

Comment:
Please add global flags to regexps. Without them only one space in each string will be replaced.